topo_processor/util/tests/aws_files_test.py

```diff
@@ -0,0 +1,25 @@
+import shutil
+from tempfile import mkdtemp
+
+import pytest
+
+from topo_processor.util.aws_files import build_s3_path, load_file_content, s3_download
+
+
+@pytest.fixture(autouse=True)
+def setup():
+    """
+    This function creates a temporary directory and deletes it after each test.
+    See following link for details:
+    https://docs.pytest.org/en/stable/fixture.html#yield-fixtures-recommended
+    """
+    target = mkdtemp()
+    yield target
+    shutil.rmtree(target)
+
+
+# Add test with AWS mock
```

Comment:
I'm not sure how far we want to test those methods which call AWS via `boto3`. There is `botocore.stubber` and `moto` library that could do the trick.

Reply:
I haven't implemented that but have created a [JIRA Task](https://linzgovt.atlassian.net/browse/TDE-236).